src/actions/__tests__/cursor-actions.test.js

```diff
@@ -0,0 +1,96 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import { moveCursorToNextChunk, updateEditorCursor } from "../actions";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe("updateEditorCursor - returns correct action", () => {
+  it("should create an action to update the cursor (NO force param)", () => {
+    const [editorCursorLine, editorCursorChar] = [1, 2];
+    const expectedAction = {
+      type: "UPDATE_CURSOR",
+      editorCursorLine,
+      editorCursorChar,
+      editorCursorForceUpdate: false
+    };
+    expect(updateEditorCursor(editorCursorLine, editorCursorChar)).toEqual(
+      expectedAction
+    );
+  });
+
+  it("should create an action to update the cursor (with force param)", () => {
+    const [editorCursorLine, editorCursorChar, editorCursorForceUpdate] = [
+      1,
+      2,
+      true
+    ];
+    const expectedAction = {
+      type: "UPDATE_CURSOR",
+      editorCursorLine,
+      editorCursorChar,
+      editorCursorForceUpdate
+    };
+    expect(
+      updateEditorCursor(
+        editorCursorLine,
+        editorCursorChar,
+        editorCursorForceUpdate
+      )
+    ).toEqual(expectedAction);
+  });
+});
```

Comment:
I would personally err on taking out the above tests, I don't think testing object creation in this way provides much value: this is *very* unlikely to break in an unexpected way and the same functionality is implicitly tested by the subsequent tests.